<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoUpdaterDialog</class>
 <widget class="QDialog" name="AutoUpdaterDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automatic Updater</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Update Available</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="currentVersion">
     <property name="text">
      <string>Current Version: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="newVersion">
     <property name="text">
      <string>New Version: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Update Notes:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="updateNotes">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="downloadAndInstall">
       <property name="text">
        <string>Download and Install...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skipThisUpdate">
       <property name="text">
        <string>Skip This Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remindMeLater">
       <property name="text">
        <string>Remind Me Later</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
